readme.md

```diff
@@ -9,15 +9,15 @@ HashR allows you to build your own hash sets based on your data sources. It's a
 HashR consists of the following components:
 
 1. Importers, which are responsible for copying the source to local storage and doing any required preprocessing.
-1. Core, which takes care of extracting the content from the source using image_export.py (Plaso), caching and repository level deduplication and preparing the extracted files for the exporters.
-1. Exporters, which are responsible for exporting files, metadata and hashes to given data sinks. Currently the main exporter is the Postgres exporter.
+2. Core, which takes care of extracting the content from the source using image_export.py (Plaso), caching and repository level deduplication and preparing the extracted files for the exporters.
```

Comment:
Please revert list numbering changes, this syntax is supported by markdown (see https://www.markdownguide.org/basic-syntax/#ordered-lists) and I think it's easier to insert an element into the list this way.

Reply:
Good day. I have reverted the list numberings.